README.md

```diff
@@ -122,6 +123,23 @@ $ ocic pod create --config test/testdata/sandbox_config.json
 ```
 # ocic ctr list
 ```
+#### Recovery from errors
```

Comment:
Was broke for me this morning.. had to wipe it clean.. figured anyone using it right now needs to know. Can push to issues if preferred.